client/src/components/Elements/DownloadData/DownloadData.js

```diff
@@ -659,7 +660,7 @@ export const DownloadData = () => {
       const legendDrugs = organism === 'styphi' ? drugsST : organism === 'kpneumo' ? drugsKP : drugsNG;
       const drugClassesBars = getDrugClassesBars();
       const drugClassesFactor = Math.ceil(drugClassesBars.length / 3);
-      const genotypesFactor = Math.ceil(genotypesForFilter.length / 6);
+      const genotypesFactor = Math.ceil(genotypesForFilterSelected.length / 6);
```

Comment:
Please use a variable instead of a hard-coded value to represent the total number of organisms. Although it currently works, please add this task to the TODO list, as updating the hard-coded value every time a new organism is added will become tedious and prone to errors in the future.